IRremoteInt.h

```diff
@@ -181,17 +176,19 @@ extern volatile irparams_t irparams;
 
 #define TOPBIT 0x80000000
 
-#define NEC_BITS 32
-#define SONY_BITS 12
```

Comment:
Sony have three sizes 12,15,20 them  I removed it we can have the SONY_MIN_BITS if do you prefer .